modules/ensemble/lib/util/utils.dart

```diff
@@ -287,7 +287,7 @@ class Utils {
   }
 
   static List<String>? getListOfStrings(dynamic value) {
-    if (value is YamlList) {
+    if (value is YamlList || value is List) {
```

Comment:
@hemish11 YamlList is List, isn't it? so the multiple check is not needed.

Reply:
Umm I did take a look at the implementation of YamlList and we do need this check, so ya closing it without changing it